downstream/modules/platform/proc-central-auth-install-dependencies.adoc

```diff
@@ -0,0 +1,46 @@
+[id="proc-central-auth-dependencies"]
+
+= Install the collections and dependencies for Ansible Galaxy
+
+.Prerequisites
+
+. A supported version of either the https://access.redhat.com/product-life-cycles[Red Hat Enterprise Linux] or the https://docs.fedoraproject.org/en-US/releases/[Fedora] operating system
+
+. A supported version of https://access.redhat.com/support/policy/updates/ansible-automation-platform[Ansible]
+
+.Procedure
+
+. Install the collection for Ansible Galaxy:
++
+[listing]
+$ ansible-galaxy collection install middleware_automation.keycloak
+
+[NOTE]
+====
+This collection has two dependencies that you might also want to install depending on your network configuration. 
+
+`middleware_automation.redhat_csp_download` enables Ansible to connect to the Customer Portal to download Red Hat’s single sign-on technology.
+
+`middleware_automation.wildfly` enables Keycloak to run on top of the Wildfly application server including Red Hat JBoss, the version of Wildfly officially supported by Red Hat.
```

Comment:
I would check that Red Hat JBoss is  the official designation.

Reply:
It is!